### PR TITLE
docs: updated getting started ios [ci skip]

### PR DESCRIPTION
**Pull request by gkartalis**
This PR resolves []

### Description

Updated the docs for opening xcode in the correct file now that the structure changed with the new app shell.

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- updated getting started docs ios - gkartalis

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: docs/generate_a_release_build_for_local_testing.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Sometimes we need to test a release build on the emulator or our local device, w
 - Start the packager with `yarn start`.
 - In another terminal, run `yarn bundle:ios` to generate the jsbundle.
 - Run `yarn pod-install`, so that the jsbundle is added to your build.
-- Open Eigen workspace project in Xcode `open Artsy.xcworkspace`.
+- Open Eigen workspace project in Xcode `open ios/Artsy.xcworkspace`.
```
Comment (pvinis) [inline docs/generate_a_release_build_for_local_testing.md:12]: good catch

File: docs/getting_started.md [hunk 2]
```diff
@@ -37,25 +37,27 @@ cd eigen
...
-  yarn setup:artsy
-  yarn install:all
-  ```
-  You will need [awscli](https://formulae.brew.sh/formula/awscli) to get our ENV vars.
+
+```
+yarn setup:artsy
+yarn install:all
+```
+
```
[the remark this answers is not in the document]
Comment (gkartalis) [reply docs/getting_started.md:47]: 😂 and the thing is that all these changes with the spacing, new lines etc come from a vscode markdown linting extension that I have

**Comment by patrinoua** [comment]
Thanks for showing our docs some love! 🎉

File: docs/getting_started.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pvinis) [reply docs/getting_started.md:47]: the newlines etc are fine. better than leaving spaces, which is a weird markdown choice. but yea.. git has a hard time sometimes :p

**Pull request merged**